crates/analyzer/src/handlers/check_clock_reset.rs

```diff
@@ -247,4 +249,25 @@ impl<'a> VerylGrammarTrait for CheckClockReset<'a> {
         }
         Ok(())
     }
+
+    fn assignment(&mut self, arg: &Assignment) -> Result<(), ParolError> {
+        if let HandlerPoint::Before = self.point {
+            if self.in_if_reset {
+                // Check to see right hand side of reset is const evaluable
+                match self.evaluator.expression(&arg.expression) {
+                    crate::evaluator::Evaluated::UnknownStatic
```

Comment:
This may be bit redundant.
How about adding `use crate::evaluator::Evaluated` to the top of the source, and removing `crate::evaluator` of this match arms?